### PR TITLE
gallery: after setActive, current item will be centered

### PR DESCRIPTION
**Pull request by venmax**
**Merge request checklist**

-   [x] All lints and tests pass. If needed, new unit tests were added.
-   [ ] If needed, the [documentation](https://github.com/mistic100/Photo-Sphere-Viewer/tree/main/docs) has been updated.

### CODE REVIEW

**Comment by mistic100** [comment]
What is the problem with scrollIntoView ? what does "alyout change" means ?

**Comment by venmax** [comment]
> What is the problem with scrollIntoView ? what does "alayout change" means ?

Its shifted the viewer out of its container.
screenshot

I don't use notification or pannel, but elements check show there psv-panel with "width:400px", and so does the notification.

File: packages/gallery-plugin/src/GalleryComponent.ts [hunk 1]
```diff
@@ -160,8 +160,13 @@ export class GalleryComponent extends AbstractComponent {
         currentActive?.classList.remove(ACTIVE_CLASS);
 
         if (id) {
-            const nextActive = this.items.querySelector(`[data-${GALLERY_ITEM_DATA_KEY}="${id}"]`);
-            nextActive?.classList.add(ACTIVE_CLASS);
+            const nextActive = this.items.querySelector(`[data-${GALLERY_ITEM_DATA_KEY}="${id}"]`) as HTMLElement;
+            if (nextActive) {
+                nextActive.classList.add(ACTIVE_CLASS);
+                // nextActive.scrollIntoView({ behavior: 'smooth', block: 'nearest', inline: 'start' });
```
Comment (mistic100) [inline packages/gallery-plugin/src/GalleryComponent.ts:166]:
```suggestion
```

File: packages/gallery-plugin/src/GalleryComponent.ts [hunk 2]
```diff
@@ -160,8 +160,13 @@ export class GalleryComponent extends AbstractComponent {
...
+                this.items.parentElement.scrollLeft =
+                    nextActive.offsetLeft + nextActive.clientWidth / 2 - this.items.parentElement.clientWidth / 2;
```
Comment (mistic100) [inline packages/gallery-plugin/src/GalleryComponent.ts:168]:
```suggestion
                this.items.scrollLeft = nextActive.offsetLeft + nextActive.clientWidth / 2 - this.items.clientWidth / 2;
```

the scrolling element changed in this commit https://github.com/mistic100/Photo-Sphere-Viewer/commit/69380e4709aa63abfc408da05c37f664563c8a89 (version 5.4.3)

**Review by mistic100 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**